less/core/accessibility.less

```diff
@@ -3,33 +3,28 @@
 .component { position: relative; }
 .block { position: relative; }
 
```

Comment:
Similar to the bootstrap example, is it worth adding a comment to instruct developers on its usage.

`// Hide content visually while keeping it accessible to assistive technologies`

Reply:
Done